### ISSUE TITLE
TranscodingStreams API

### ISSUE THREAD

**Issue by JonasIsensee** [JuliaIO/Blosc.jl#79]
Hi,

I'm currently working on bringing better compression to [JLD2.(https://github.com/JuliaIO/JLD2.jl).
Most of the other compression libraries seem to support the [TranscodingStreams](https://github.com/JuliaIO/TranscodingStreams.jl) API and it appears to be straight forward to utilize that for JLD2.

At this time `Blosc` is not compatible to `TranscodingStreams` but it would be much easier to enable it within JLD2 if it were.
Would it be possible to add the API functions? Has this been discussed before?

**Comment by stevengj** [comment]
I don't think this has been discussed before but it would be reasonable to add.

**Comment by JonasIsensee** [comment]
Here's the progress on the JLD2 side:

https://github.com/JuliaIO/JLD2.jl/pull/264
